pxr/imaging/hd/sceneDelegate.h

```diff
@@ -191,8 +191,7 @@ struct HdPrimvarDescriptor {
         : name(name_), interpolation(interp_), role(role_), indexed(indexed_)
     { }
     bool operator==(HdPrimvarDescriptor const& rhs) const {
```

Comment:
we'll want to add a test, similar to what @nvidia-jomiller did in https://github.com/PixarAnimationStudios/USD/pull/1807, that fails without the change, and passes with it. Thanks!